mm0-rs/README.md

```diff
@@ -6,19 +6,79 @@ Watch this space for more updates, as `mm0-rs` is under active development. See
 
 ## Compilation
 
-To compile `mm0-rs`, run
+To compile `mm0-rs`, first install Rust.
+You then need to download this repository (e.g.,
+`git clone https://github.com/digama0/mm0.git`).
+Then cd into this `mm0-rs` directory (using `cd mm0/mm0-rs`) and run:
 
     cargo build --release
 
-from the `mm0-rs` directory, then copy or symlink the resulting executable file `target/release/mm0-rs` to your system path and/or point `vscode-mm0` to it using the setting
+You then should make it easy to execute.
+One way is to copy or symlink the resulting executable file `target/release/mm0-rs` to your system path.
+An alternative, if you're integrating with Visual Studio Code as suggested
+below, is to point `vscode-mm0` to the executable using (as
+explained below) the setting "metamath-zero.executablePath": "mm0-rs"
+in your `settings.json` file.
 
-    "metamath-zero.executablePath": "mm0-rs",
+## Integration with Visual Studio Code
 
-in your `settings.json` file.
+We typically use Visual Studio Code, an open source software
+editor / IDE which supports LSP servers.
+If you choose to do the same,
+[download and install Visual Studio Code](https://code.visualstudio.com/).
+
+Select View/Extensions, search for "Metamath"
+Look for "Metamath Zero" & press its "install" button.
```

Comment:
```suggestion
Then use Quick Open (Ctrl-P), paste `ext install digama0.metamath-zero` in the box and hit enter, which will install the ["Metamath Zero" language support](https://marketplace.visualstudio.com/items?itemName=digama0.metamath-zero) extension.
```
Better than asking people to search for a thing and install the first thing they find.

Reply:
Control-P doesn't work on MacOS, and I think that's way more complicated than using View/Extensions. Perhaps describe it both ways?

Reply:
If there are multiple versions I think people will notice :-).

Reply:
Well you can just click the link to go to vscode marketplace, which has instructions appropriate for the OS. Probably we'd best just do that.

Reply:
(Command-P is the macOS counterpart of Control-P)